cmake/FindLibBpf.cmake

```diff
@@ -3,12 +3,12 @@
 #
 #  LIBBPF_FOUND            - system has libbpf
 #  LIBBPF_INCLUDE_DIR      - the libbpf include directory
-#  LIBBPF_SOURCE_DIR       - the libbpf source directory
+#  LIBBPF_STATIC_LIBRARIES - the libbpf source directory
 #  LIBBPF_LIBRARIES        - link these to use libbpf
 
-#if (LIBBPF_LIBRARIES AND LIBBPF_INCLUDE_DIR AND LIBBPF_SOURCE_DIR)
+#if (LIBBPF_LIBRARIES AND LIBBPF_INCLUDE_DIR AND LIBBPF_STATIC_LIBRARIES)
 #  set (LibBpf_FIND_QUIETLY TRUE)
-#endif (LIBBPF_LIBRARIES AND LIBBPF_INCLUDE_DIR AND LIBBPF_SOURCE_DIR)
+#endif (LIBBPF_LIBRARIES AND LIBBPF_INCLUDE_DIR AND LIBBPF_STATIC_LIBRARIES)
 
```

Comment:
Looks like it should work. Will give a try later.
Could you remove the libbpf-debugsource comments below as well?

Reply:
done, thanks